substrate/utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -748,13 +748,16 @@ impl CliConfiguration for PalletCmd {
 
 /// List the benchmarks available in the runtime, in a CSV friendly format.
 fn list_benchmark(
-	benchmarks_to_run: Vec<(
+	mut benchmarks_to_run: Vec<(
 		Vec<u8>,
 		Vec<u8>,
 		Vec<(BenchmarkParameter, u32, u32)>,
 		Vec<(String, String)>,
 	)>,
 ) {
+	// Sort by pallet and function name.
+	benchmarks_to_run.sort_by(|(pa, sa, _, _), (pb, sb, _, _)| (pa, sa).cmp(&(pb, sb)));
```

Comment:
What about multiple pallet instances e.g. pallet_assets?
Shouldnt we consider also `BenchmarkList.instance` attribute?

Reply:
It is currently not possible to benchmark only a single instance of a pallet. And IMHO it does not make much sense.  
But they will be listed multiple times, i will add a dedub to it.

Reply:
ok then,
I know the only one place where `list` command is used: https://github.com/paritytech/command-bot-scripts/blob/main/commands/bench/lib/bench-all-cumulus.sh#L26-L34 but I dont know what is your use case here :)

Reply:
Yea that also has a `uniq` in the pipe to dedub the instances, but now we already do it here.